### PR TITLE
Remove footer

### LINKED ISSUE #1775: remove footer; change Rust menu to "About"
Right now docs.rs adds a footer as well as a header to each page. This is a frequent source of layout bugs, as it interacts in a complicated way with rustdoc's CSS. It contains:

- [About docs.rs](https://docs.rs/about)
- [Privacy policy](https://foundation.rust-lang.org/policies/privacy-policy/#docs.rs)
- [Build queue](https://docs.rs/releases/queue)

I propose that we eliminate the footer, and move these links to the topbar. If we rename the "Rust" menu to "About", I think they would fit well there. This would also make them more visible and easier to reach. I think many people never notice the footer all the way at the bottom of the page (I didn't for years, until I saw a bug report about it :-D).

### PR DESCRIPTION
**Pull request by jsha**
Move "About" and "Privacy policy" into the Rust menu.

The footer required a lot of workarounds to work with the various bits of HTML that rustdoc (and doc authors!) generated. It's hard enough to make the topbar work consistently; adding a footer to nearly-arbitrary HTML is too much, particularly when we can achieve the same thing in the topbar.

Fixes #1775.

image

### CODE REVIEW

File: templates/footer.html [hunk 1]
```diff
@@ -1,5 +0,0 @@
-<div class="docs-rs-footer">
```
Comment (GuillaumeGomez) [inline templates/footer.html:1]: Strangely enough, this file is not marked as deleted. Github UI bug?

**Comment by GuillaumeGomez** [comment]
This is a great idea!

File: templates/footer.html (hunk 1 above)
Comment (jsha) [reply templates/footer.html:1]: Nope, not a GitHub UI bug - I had deleted the contents but not the file. In my most recent push it is now properly deleted.

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
It'll make things so much simpler!

**Pull request merged**